cmd/cloud/installation.go

```diff
@@ -34,6 +34,8 @@ func init() {
 	installationDeleteCmd.MarkFlagRequired("installation")
 
 	installationGetCmd.Flags().String("installation", "", "The id of the installation to be fetched.")
+	installationGetCmd.Flags().Bool("include-group-config", false, "Whether to include group configuration in the installation or not.")
```

Comment:
I agree that we would want the defaults to be whatever the provisioner is actually using to make updates. Because groups are currently still a no-op I left that info out. I will change the defaults when group values start impacting the installations.